src/server/examples/examples_test.go

```diff
@@ -341,3 +342,107 @@ func TestFruitStand(t *testing.T) {
 	require.NoError(t, c.DeleteRepo("filter", false))
 	require.NoError(t, c.DeleteRepo("data", false))
 }
+
+func TestScraper(t *testing.T) {
+	if testing.Short() {
+		t.Skip("Skipping integration tests in short mode")
+	}
+	c := getPachClient(t)
+
+	// Check that scraper pipeline manifest appears in the docs (before we create a new repo or anything)
+	readme, err := ioutil.ReadFile("../../../doc/examples/scraper/README.md")
```

Comment:
Fixed. I just got rid of this section (I don't pass this string to pachyderm, I was just using the file that's already in this section)